src/index.js

```diff
@@ -11,13 +11,17 @@ module.exports = function emberCliUpdate(options) {
   let from = options.from;
   let to = options.to;
   let ignoreConflicts = options.ignoreConflicts;
+  let packageVersion = getPackageVersion('.');
+
+  if (!packageVersion) {
+    return new Promise((_, reject) => reject('Ember CLI was not found in this project\'s package.json'));
```

Comment:
This can be `Promise.reject('Ember CLI was not found in this project\'s package.json')` like https://github.com/kellyselden/git-diff-apply/blob/v0.3.1/src/index.js#L25

Reply:
Done